pt/utils/verify.py

```diff
@@ -0,0 +1,13 @@
+import json
+import hashlib
+import base64
+import requests
+from config import APP as app
+
```

Comment:
You can put a blank line between  `import request` &  `from config import ...`

Reply:
Fix in 530a660.